cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/saved_search.spec.js

```diff
@@ -90,14 +92,20 @@ const runSavedSearchTests = () => {
 
       it(`should successfully update a saved search for ${config.testName}`, () => {
         // using a POST request to create a saved search to load
-        postRequestSaveSearch(config);
-        updateSavedSearchAndSaveAndVerify(config, workspaceName, DATASOURCE_NAME, false);
+        cy.wrap(null).then(() => {
```

Comment:
Is this to ensure that the async operations are completed before continuing to the assertions?